x-pack/plugins/security_solution/public/resolver/view/panels/descriptive_name.tsx

```diff
@@ -55,6 +55,18 @@ export function DescriptiveName({ event }: { event: SafeResolverEvent }) {
     );
   }
 
+  if (event.kibana?.alert?.rule?.name) {
+    return (
+      <FormattedMessage
+        id="xpack.securitySolution.resolver.eventDescription.alertEventNameLabel"
+        defaultMessage="{ ruleName }"
+        values={{
+          ruleName: String(event.kibana?.alert?.rule?.name),
```

Comment:
Even though it should be there this would lead to 'undefined' or 'null' if it isn't there and wouldn't it already be a string? So maybe `event.kibana?.alert?.rule?.name ?? ''`;

Reply:
yeah agreed, i was following the pattern of the rest of the file is all